### PR TITLE
Added Profile View Screen

### LINKED ISSUE #5: Create a profile view containing Person information
Simply use relevant fields (add some if necessary) of the Person object and then show them on the Profile view. We will connect this to firebase later on.

### PR DESCRIPTION
**Pull request by sacchit25**
Added Profile View Screen

Fixes #5

https://github.com/user-attachments/assets/6866af1d-5317-4993-9367-828921215732

### CODE REVIEW

File: Vineyard/Vineyard/ContentView.swift [hunk 1]
```diff
@@ -12,7 +12,18 @@ struct ContentView: View {
     var body: some View {
         NavigationStack {
             VStack {
-                Text("\(viewModel.user.name)")
+                HStack {
```
Comment (sankaet8) [inline Vineyard/Vineyard/ContentView.swift]: I would recommend switching this from an HStack to a toolbar. This is a good site which holds all that info: https://holyswift.app/how-to-create-toolbar-in-swiftui/.

**Review by sankaet8 — CHANGES_REQUESTED** [review]
Really good stuff! Once you change the HStack to a toolbar we can merge this in :)

File: Vineyard/Vineyard/ContentView.swift (hunk 1 above)
Comment (sacchit25) [reply Vineyard/Vineyard/ContentView.swift]: Replaced. Is that what you were looking for?

**Review by sankaet8 — CHANGES_REQUESTED** [review]
Yes that looks good! Also one additional note, if you are adding and removing files from the project, like the ProfileView.swift, please also commit the project.pbxproj file. So if you could do that too, that would be perfect :)

**Pull request closed without merging**